### DISCUSSION TITLE
Tests interrupteurs Aqara H1 et binding - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#40] [Comments]
# Tests interrupteurs Aqara H1 et binding - Haade.fr

test de l’interrupteur aqara h1 version europe avec télécommande h1 et initiation à la liaison zigbee bindings

https://haade.fr/fr/blog/test-xiaomi-aqara-interrupteur-h1-europe-et-lien-binding

**Comment by Godinouu** [comment]
Hello Nico et merci pour ce test fouillé, ça fait du bien de lire des arguments techniques qui sont rarement mis en avant ailleurs ! Au fond la plus grosse difficulté est de mettre un inter virtuel sur un physique pour éviter la mise hors tension du luminaire zigbee tout en laissant le traditionnel inter en place !

**Reply by haade-administrator** [reply]
Merci, ça fait toujours plaisir, mais dès fois il est difficile de trouver des arguments pertinents sur les produits testés.
Laisser l'inter traditionnel en place et le domotiser sur la face arrière est une solution, à chacun de voir la façon de domotiser l'éclairage.

**Comment by Lulu09320** [comment]
Bonjour et merci pour ce test très poussé ! ChatGPT m'a fait découvrir ce site et je l'ai directement ajouté à mes favoris ! Le travail a l'air d'une qualité remarquable, merci ! 
Je cherche des interrupteurs zigbee compatible binding (pour assurer une tolérance de panne en cas d'indisponibilité de mon HomeAssistant / Z2M). Mais je cherche un truc un peu esthétique qui ne coute pas la peau des fesses... (j'ai toute une maison à équiper). Ceux-là paraissent bien sur le papier mais c'est très dommage que le binding ne fonctionne qu'à moitié...
Avez-vous pu tester avec le bridge Xiaomi pour voir si une MAJ était sortie ? Peut-être côté Z2M vu que l'article date d'il y a quelques mois ?
Après je me dis qu'en secours, si 50% de mes lumières s'allument c'est suffisant. L'idée étant de ne pas être dans le noir

**Reply by haade-administrator** [reply]
Merci,
Pour le moment Z2M n'a pas évolué sur le sujet.
Pour le bridge Xiaomi, non je n'ai pas testé avec mais ça devrait fonctionner sinon ils n'en feraient pas la promotion
Le binding est une part très importante dans la domotique et pourtant on en entend pas parler, il est fonctionnel si les modules testés ont la même puce intégrée même si on en parle pas sur les forums.
Les modules à puces tuya tu peux oublier
Si tu veux faire du binding passe sur du z-wave, les tarifs sont devenus très intéressant et tous les modules sont compatibles entre eux de n'importe quelle marque et génération
